tests/storage/blocksd_test.py

```diff
@@ -29,6 +29,7 @@
 from vdsm.storage import sanlock_direct
 from vdsm.storage import sd
 from vdsm.storage import sp
+from vdsm.storage.volume import Qcow2BitmapInfo
 from vdsm.storage.sdc import sdCache
 from vdsm.storage.sdm.api import merge as api_merge
 from vdsm.storage.spbackends import StoragePoolDiskBackend
```

Comment:
We try to keep alphabetical order within import blocks.

This block is a little messed up with the more/less specific imports.
So maybe you could move the `volume` module import after the `spbackends` module.